### PR TITLE
Fix static docs for 2.0 changes

### PR DESCRIPTION
**Pull request by suyograo**

### CODE REVIEW

File: docs/asciidoc/static/advanced-pipeline.asciidoc [hunk 1]
```diff
@@ -135,14 +135,12 @@ Elasticsearch cluster. Edit the `first-pipeline.conf` file to add the following
...
     }
 }
 
-With this configuration, Logstash uses multicast discovery to connect to Elasticsearch. 
-
-NOTE: Multicast discovery is acceptable for development work, but unsuited for production environments. For the 
-purposes of this example, however, the default behavior is sufficient.
+With this configuration, Logstash uses http protocol to connect to Elasticsearch. The above example assumes Logstash
+and Elasticsearch to be running on the same instance. You can specify a remote Elasticsearch instance using `hosts`
+configuration like `hosts => es-machine:9092`.
```
Comment (untergeek) [inline docs/asciidoc/static/advanced-pipeline.asciidoc]: For consistency's sake, should we have the host:port in quotes?

```
hosts => "es-machine:9092"
```

File: docs/asciidoc/static/advanced-pipeline.asciidoc [hunk 2]
```diff
@@ -430,14 +426,14 @@ To configure your Logstash instance to write to multiple Elasticsearch nodes, ed
...
+        hosts => ["IP Address 1:port1", "IP Address 2:port2", "IP Address 3"]
     }
 }
 --------------------------------------------------------------------------------
 
-Use the IP addresses of three non-master nodes in your Elasticsearch cluster in the host line. When the `host` 
-parameter lists multiple IP addresses, Logstash load-balances requests across the list of addresses.
+Use the IP addresses of three non-master nodes in your Elasticsearch cluster in the host line. When the `hosts` 
+parameter lists multiple IP addresses, Logstash load-balances requests across the list of addresses. Also note that
+default port for Elasticsearch is `9092` and can be omitted in the configuration above.
```
Comment (untergeek) [inline docs/asciidoc/static/advanced-pipeline.asciidoc]: Did it change from 9200?

**Comment by untergeek** [comment]
Other than my notes, LGTM™

**Comment by suyograo** [comment]
thanks @untergeek. Updated

**Comment by untergeek** [comment]
@suyograo LGTM™

**Pull request closed without merging**